.github/workflows/prod-api-CI.yml

```
@@ -0,0 +1,35 @@
+    paths:
+      - "api-server/**"
+      - "common-module/**"
+      - "core-domain/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    environment: prod-api
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)